### PR TITLE
Kadir-TDD-Part1&part2

### PR DESCRIPTION
**Pull request by laperla11**

### CODE REVIEW

File: II.pass-tests/08-get-average/get-average.js [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+module.exports = function (numbers) {
+  //filtering the numbers
+  numbers = numbers.filter(number => {
+    return typeof number === "number";
+  });
+  return (numbers.reduce((total, number) => {
+    //finding the total of filtered numbers
+    return total + number;
+  }) / numbers.length // calculating the average
```
Comment (40thieves) [inline II.pass-tests/08-get-average/get-average.js]: Nice usage of comments :)

File: II.pass-tests/ES-6-practice/es6.js [hunk 2]
```diff
@@ -1,27 +1,27 @@
 // Turn this function into an arrow function
-function arrow() {
+const arrow = () => {
```
Comment (40thieves) [inline II.pass-tests/ES-6-practice/es6.js]: Oh hmm, I guess we forgot about these. We're going to go over ES6 more tomorrow

File: I.write-tests/06-largest-number/largest-number.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 function getLargestNumber(array) {
-    var largestNumber;
-    for (var i = 0; i < array.length - 1; i++) {
-        if (array[i] > array[i + 1]) {
-            largestNumber = array[i]
-        }
+  var largestNumber = -Infinity;
```
[the remark this answers is not in the document]
Comment (laperla11) [reply I.write-tests/06-largest-number/largest-number.js]: I do not use that assignment  for the case no numbers are given. I assigned negative infinity to the `largestNumber` just before the loop to compare with the first number with that. In this way, first number always become the first largest number after that I am comparing the following numbers with the new largest number. If I do not assign any value in the beginning the function returns undefined.

File: I.write-tests/02-remove-vowels/remove-vowels.test.js [hunk 4]
```diff
@@ -1,8 +1,19 @@
...
+  const expectation1 = "mr"
+  const expectation2 = "kdr"
+  // Act
+  var result = removeVowels(name);
+  var result1 = removeVowels(name1)
+  var result2 = removeVowels(name2)
+  //Assert
+  expect(result).toEqual(expectation);
+  expect(result1).toEqual(expectation1);
+  expect(result2).toEqual(expectation2);
```
[the remark this answers is not in the document]
Comment (laperla11) [reply I.write-tests/02-remove-vowels/remove-vowels.test.js]: Thank you very much for your comments [40thieves](/40thieves).
I am also trying to solve the katas If I have time. Trying to solve katas helps me to remember what I've learnt so far. It also pushes me to search for new things and learn a lot.

File: II.pass-tests/04-find-needle/find-needle.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
-function findNeedle(words) {
-
+module.exports = function (words, wordToFindIndex) {
```
[the remark this answers is not in the document]
Comment (laperla11) [reply II.pass-tests/04-find-needle/find-needle.js]: Thanks for explanation the I fixed it.

**Pull request closed without merging**